qiita_db/analysis.py

```diff
@@ -1011,6 +1011,18 @@ def _build_mapping_file(self, samples, rename_dup_samples=False):
                     samps = set(samps) - all_ids
                     all_ids.update(samps)
 
+                # appending study metadata to the analysis
+                study = qdb.artifact.Artifact(aid).study
+                study_owner = study.owner
+                study_info = study.info
+                pi = study_info['principal_investigator']
+                qm['qiita_study_title'] = study.title
+                qm['qiita_study_alias'] = study.info['study_alias']
+                qm['qiita_owner'] = '%s (%s)' % (study_owner.info['name'],
+                                                 study_owner.email)
+                qm['qiita_principal_investigator'] = '%s (%s)' % (
+                    pi.name, pi.email)
```

Comment:
If we include the email, then we are absolutely including the `@` symbol, which is an unsupported character as per the QIIME mapping file description, I think this is fine with just the PI's name.